modules/serverless-tech-preview-features.adoc

```diff
@@ -15,52 +15,61 @@ ifdef::openshift-enterprise,openshift-dedicated[]
 .Generally Available and Technology Preview features tracker
 [cols="3,1,1,1",options="header"]
```

Comment:
```suggestion
[cols="4,1,1,1",options="header"]
```

I'm not 100% sure but the number of columns is 4 now. So, you need to update it.
Otherwise, the table is broken right now like [here](https://53365--docspreview.netlify.app/openshift-enterprise/latest/serverless/serverless-release-notes.html#serverless-tech-preview-features_serverless-release-notes):

![image](https://user-images.githubusercontent.com/2138339/205230537-46472e83-160b-4564-8708-4e9f036f9bd3.png)

Please see `1.26` is in the wrong position.